charge-holy-weapon.lic

```diff
@@ -26,24 +27,34 @@ class HolyWeapon
   end
 
   def check_charge_level(hw_settings)
-    @em.wield_weapon(hw_settings['weapon_name'])
+    weapon = hw_settings['weapon_name']
+    threshold = hw_settings['power_level_threshold'] || 3
```

Comment:
Why default to 3 and not 4 like previous?

Reply:
That was a typo, I thought 3 did refer to the the old power level value, thanks. It shouldn't have changed the behavior from the original script.